### PR TITLE
update code owners

### PR DESCRIPTION
**Pull request by anna-carroll**
## Motivation
code owners weren't super granular / accurate

## Solution
took a stab at updating & narrowing down the user groups

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,14 +1,22 @@
...
-packages/contracts-bridge/ @nomad-xyz/experience @nomad-xyz/protocol
-packages/deploy/ @nomad-xyz/experience @nomad-xyz/protocol
-packages/multi-provider/ @nomad-xyz/experience @nomad-xyz/protocol
-packages/contracts-router/ @nomad-xyz/experience @nomad-xyz/protocol
-
-# SDK Packages 
-packages/sdk/ @nomad-xyz/experience @nomad-xyz/protocol 
-packages/sdk-govern/ @nomad-xyz/experience @nomad-xyz/protocol
-packages/sdk-bridge/ @nomad-xyz/experience @nomad-xyz/protocol
+# Contracts packages
```
Comment (anna-carroll) [inline .github/CODEOWNERS:4]: `Contracts packages`  are pretty obvious / uncontroversial - should be @nomad-xyz/protocol

File: .github/CODEOWNERS [hunk 2]
```diff
@@ -1,14 +1,22 @@
...
+packages/upgrade/ @nomad-xyz/protocol
+
+# SDK Packages
+packages/multi-provider/ @nomad-xyz/experience @prestwich
+packages/sdk/ @nomad-xyz/experience
+packages/sdk-govern/ @nomad-xyz/experience
+packages/sdk-bridge/ @nomad-xyz/experience
+
+# DevOps Packages
+packages/keymaster/ @devops
```
Comment (prestwich) [inline .github/CODEOWNERS:18]: github flagging as unknown

Comment (anna-carroll) [reply .github/CODEOWNERS]: thanks, fixed: [fix: nomad-xyz/devops](https://github.com/nomad-xyz/monorepo/pull/470/commits/888541d16372a634f72733875d4ef45ee830f7fe)

Comment (Imti) [reply .github/CODEOWNERS]: `@nomad-xyz/devops`?

File: .github/CODEOWNERS [hunk 3]
```diff
@@ -1,14 +1,22 @@
...
-packages/sdk-govern/ @nomad-xyz/experience @nomad-xyz/protocol
-packages/sdk-bridge/ @nomad-xyz/experience @nomad-xyz/protocol
+# Contracts packages
+packages/contracts-bridge/ @nomad-xyz/protocol
+packages/contracts-core/ @nomad-xyz/protocol
+packages/contracts-router/ @nomad-xyz/protocol
+packages/deploy/ @nomad-xyz/protocol
+packages/upgrade/ @nomad-xyz/protocol
+
+# SDK Packages
```
[the remark this answers is not in the document]
Comment (Imti) [reply .github/CODEOWNERS:11]: This is good for now

File: .github/CODEOWNERS [hunk 4]
```diff
@@ -1,14 +1,22 @@
...
+packages/multi-provider/ @nomad-xyz/experience @prestwich
+packages/sdk/ @nomad-xyz/experience
+packages/sdk-govern/ @nomad-xyz/experience
+packages/sdk-bridge/ @nomad-xyz/experience
+
+# DevOps Packages
+packages/keymaster/ @devops
+
+# Other Packages
```
[the remark this answers is not in the document]
Comment (Imti) [reply .github/CODEOWNERS:20]: This looks good to me 🙏

File: .github/CODEOWNERS (hunk 3 above)
[the remark this answers is not in the document]
Comment (ErinHales) [reply .github/CODEOWNERS:11]: Yes, that's good

**Review by ErinHales — APPROVED** [review]

**Review by Imti — APPROVED** [review]
🦄

**Pull request merged**